source/app.d

```diff
@@ -46,17 +49,16 @@ class ImageFitness
         destination = loadImage(path);
     }
 
-    void rasterize(ref SuperImage image, in Circle[] circles)
+    void rasterize(ref SuperImage image, in Rect[] rects)
     {
-        foreach(ref circle; circles)
+        foreach(ref rect; rects)
         {
-            auto color = circle.color;
-            foreach(x; (circle.x - circle.radius / 2) .. (circle.x + circle.radius / 2))
+            auto color = rect.color;
+            foreach(x; (rect.x) .. (rect.x + rect.width))
```

Comment:
Já vím, že to tam bylo předtím taky, ale fakt není třeba dávat ty hodnoty do závorek.

Stejně tak bych dal mezeru za ty foreache, viz https://dlang.org/dstyle.html.